docker-compose.test.yml

```diff
@@ -0,0 +1,14 @@
+version: '2.1'
```

Comment:
I think there is version `3` already.

Reply:
Yes, I used the most recent one that is compatible with docker+compose releases used in production.